x-pack/plugins/cloud_security_posture/public/pages/findings/findings_table.test.tsx

```diff
@@ -23,7 +23,7 @@ const getFakeFindings = (): CspFinding & { id: string } => ({
     },
   },
   rule: {
-    name: chance.word(),
+    name: chance.sentence(),
```

Comment:
using `word()` was wrong, it's really a sentence anyway. the test failed because 2 rules had the same name - a 2 char word.